src/org/zaproxy/zap/extension/codedx/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Code Dx Extension</name>
-	<version>5</version>
+	<version>5.0.1</version>
```

Comment:
I'm pretty sure versions have to be integers.
We do support semantic versioning as well (eg `<semver>2.0.0</semver>`) but I think we still need the integer version - that right @thc202 ?

Reply:
The `changes` element should be updated too.